app/src/main/java/fr/free/nrw/commons/contributions/ContributionsListFragment.java

```diff
@@ -123,10 +102,8 @@ private void initializeAnimations() {
 
     private void setListeners() {
         fabPlus.setOnClickListener(view -> animateFAB(isFabOpen));
-        fabCamera.setOnClickListener(view -> {
-            controller.initiateCameraPick(getActivity());
-        });
-        fabGallery.setOnClickListener(view -> controller.initiateGalleryPick(getActivity()));
+        fabCamera.setOnClickListener(view -> controller.initiateCameraPick(getActivity(), CAMERA_UPLOAD_REQUEST_CODE));
+        fabGallery.setOnClickListener(view -> controller.initiateGalleryPick(getActivity(), 5, GALLERY_UPLOAD_REQUEST_CODE));
```

Comment:
Would be great if you could extract this 5 to a constant field, will be much easier to understand the param.